VimeoUpload/Descriptor System/DescriptorManagerArchiver.swift

```diff
@@ -35,7 +35,7 @@ class DescriptorManagerArchiver
 
     // MARK: 
     
-    private let archiver: KeyedArchiver
+    private let archiver: KeyedArchiver?
```

Comment:
Does it make sense to instantiate the `DescriptorManagerArchiver` without the `archiver`? It looks like we want this guy non optional. We prolly want the initializer being failable?